src/theme/default/index.js

```diff
@@ -1,11 +1,33 @@
 export const defaultTheme = {
-  breakpoints: ['40em', '52em', '64em'],
-  fontSizes: [12, 14, 16, 20, 24, 32, 48, 64],
+  breakpoints: {
+    sm: '40em',
+    md: '52em',
+    lg: '64em',
+  },
+  fontSizes: {
+    xxxl: '44px',
```

Comment:
muze bit drzet jako `number`? 
```suggestion
    xxxl: 44,
```

Reply:
Máš to tam.